blocklist-client/src/main.rs

```diff
@@ -0,0 +1,41 @@
+use crate::config::Settings;
+use ::config::{Config, File};
```

Comment:
Isn't these both the same import? I think we should be explicit and go with `crate::config` for both, and avoid relative imports.

Reply:
First one is for the config.rs at root level. Second one is from a lib. 
That's what the auto-import gives and for some reason it's not happy when I don't use :: for the second one.